Mcorrea update

portermahoney opened the pull request:
Updating based on hers

portermahoney committed:
update commit hashes in Dockerfiles

portermahoney committed:
update based on hers and fiine

aaronk commented on .gitignore:
```diff
@@ -18,10 +19,14 @@ __pycache__
 docker-compose-local.yml
-# Byte-compiled / optimized / DLL files
+*.swp
+yarm.lock
```
yarn.lock

aaronk commented on Dockerfile:
```diff
@@ -11,24 +11,27 @@ RUN mkdir ~/.ssh && echo "Host git*\n\tStrictHostKeyChecking no\n" >> ~/.ssh/con
 RUN echo "daemon off;" >> /etc/nginx/nginx.conf
 COPY etc/nginx.conf /etc/nginx/sites-available/default
 COPY etc/supervisor.conf /etc/supervisor/conf.d/app.conf
+COPY etc/logging.ini /etc/logging.ini
```
Remove this.  This never really worked.

aaronk commented on Dockerfile-drf:
```diff
@@ -28,15 +30,19 @@ RUN --mount=type=ssh pip install --upgrade pip && \
+    pip install git+ssh://git@gitlab-int.rc.fas.harvard.edu/informatics/ifxvalidcode.git@${IFXVALIDCODE_COMMIT} && \
     pip install -r requirements.txt
 
+COPY etc/logging.ini /etc/logging.ini
```
Remove

aaronk commented on README.md:
```diff
@@ -9,7 +9,7 @@ processed (project_name substituted) are listed by name and extension:
     > mkdir ifxtest
     > cd ifxtest
-    > django-admin startproject --template=https://github.com/harvardinformatics/ifxdjango/archive/v2.0.zip -e py,html,js,conf,sh,json,cfg,txt,vue -n .env.development,.env.production,Dockerfile-drf,Dockerfile-ui,Dockerfile,docker-compose.yml,Makefile ifxtest .
+    > django-admin startproject --template=https://github.com/harvardinformatics/ifxdjango/archive/v2.0.zip -e py,html,js,conf,sh,json,cfg,txt,vue -n .env.development,.env.production,Dockerfile-drf,Dockerfile-ui,Dockerfile,docker-compose.yml,Makefile,logging.ini,cypress.json,commands.js,models.py,admin.py,permissions.py,serializers.py,data.py,settings.py,processor.py ifxtest .
```
js, py, json all taken care of by -e

aaronk commented on etc/logging.ini:
```diff
@@ -0,0 +1,30 @@
+
```
Kill this whole file

aaronk commented on project_name/request/views.py:
```diff
@@ -1,10 +1,9 @@
 # -*- coding: utf-8 -*-
 '''
-views for {{project_name}}.request
+views for hers.request
```
Revert to {{project_name}}

aaronk commented on project_name/settings.py:
```diff
@@ -305,16 +310,3 @@ class IFXMESSAGES():
     ACCOUNT_REQUEST_COMPLETED_MESSAGE = '{{project_name}}_account_request_completed_message'
     ACCOUNT_REQUEST_COMPLETED_USER_NOTIFICATION_MESSAGE = '{{project_name}}_account_request_completed_user_notification_message'
 
-# Assuming ifxbilling is installed, these models should be ignored by the django-author pre-save
```
This section needs to remain

aaronk commented on requirements.txt:
```diff
@@ -1,12 +1,13 @@
 Django>2.2, <3
 django-nose==1.4.5
-djangorestframework>3.9.1<3.12.0
+djangorestframework>3.9.1
```
I'm pretty sure we need to leave the <3.12.0 in there, but I could be wrong.

portermahoney committed:
update from feedback

portermahoney committed:
more corrections

aaronk committed:
update axios, lodash due to security warnings

aaronk committed:
Updated core-js to fix missing component
Using p3's Login (redirect to login component) so that a signal can be emitted
Changed Login / Logout on the app bar to be p3 like
Added id to authUser so that link to UserDetail will work

aaronk committed:
final newline

portermahoney committed:
add admin group to fixtures to run in applyDevData and fix VUE_APP_DJANGO

portermahoney committed:
fix ports

portermahoney committed:
make sure all the drf ports match, move cypress install to Dockerfile-ui and remove from package.json

portermahoney committed:
update vuetify config, fix AccountRequestDetail project_name
